pom.xml

```diff
@@ -79,7 +79,9 @@
     <cdap.version>6.1.0-SNAPSHOT</cdap.version>
     <common.codec.version>1.12</common.codec.version>
     <docs.dir>docs</docs.dir>
-    <google.cloud.storage.version>1.100.0</google.cloud.storage.version>
+    <google.cloud.storage.version>1.57.0</google.cloud.storage.version>
+    <google.cloud.vision.version>1.57.0</google.cloud.vision.version>
+    <google.protobuf.version>3.5.1</google.protobuf.version>
```

Comment:
I had problems with new protobuf because of older version in CDAP core. Can you please double check that it will not be a conflict here.

Reply:
I have no issues running the plugin on CDAP sandbox(both preview and deploy modes) and running integration tests using this local CDAP instance on Dataproc. 
CDAP built from https://github.com/cdapio/cdap/tree/a11c59668fff3649fe43b4a271364f0457da0d4e